src/features/creators/newPackageProject.ts

```diff
@@ -140,7 +140,20 @@ export class NewPackageProject implements PythonProjectCreator {
                             quickCreate: true,
                         });
                     } else {
-                        window.showErrorMessage(l10n.t('Creating virtual environment failed during package creation.'));
+                        const action = await window.showErrorMessage(
+                            l10n.t(
+                                'Could not automatically create a virtual environment for the new package "{0}" because your default environment manager does not support this operation and no alternative was available.',
```

Comment:
```suggestion
                                'A virtual environment could not be created for the new package "{0}" because your default environment manager does not support this operation and no alternative was available.',
```